sklearn/metrics/tests/test_pairwise.py

```diff
@@ -649,9 +649,9 @@ def test_euclidean_distances(dtype, x_array_constr, y_array_constr):
     # check that euclidean distances gives same result as scipy cdist
     # when X and Y != X are provided
     rng = np.random.RandomState(0)
-    X = rng.random_sample((100, 10)).astype(dtype, copy=False)
+    X = rng.random_sample((110, 10000)).astype(dtype, copy=False)
     X[X < 0.8] = 0
-    Y = rng.random_sample((10, 10)).astype(dtype, copy=False)
+    Y = rng.random_sample((100, 10000)).astype(dtype, copy=False)
```

Comment:
If the fix is about the handling of `n_samples`, why do you need to increase `n_features` so much?

Reply:
I guess it's to make the batch size small enough to test batching. Please add a comment to explain this.

Reply:
The batch size depends on the total size (number of elems) in X (resp. Y). Increasing more the number of features is a bit faster and makes the final distance matrix smaller.